tests/acm_observability/conftest.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+import pytest
+import requests
+from ocp_resources.multi_cluster_observability import MultiClusterObservability
+from ocp_resources.route import Route
+from pytest_testconfig import config as py_config
+
+
+@pytest.fixture(scope="session")
+def multi_cluster_observability(admin_client_scope_session):
+    observability = MultiClusterObservability(
+        client=admin_client_scope_session,
+        name="observability",
+    )
+    assert observability.exists
+    assert (
+        observability.instance.status.conditions[-1].type == observability.Status.READY
+    )
+
+    return observability
+
+
+@pytest.fixture(scope="session")
+def rbac_proxy_route_url(admin_client_scope_session, multi_cluster_observability):
+    rbac_proxy_route_url = Route(
+        client=admin_client_scope_session,
+        name="rbac-query-proxy",
+        namespace="open-cluster-management-observability",
```

Comment:
`multi_cluster_observability` have namespace

Reply:
`MultiClusterObservability` is cluster-scoped